docs/basics/troubleshooting.md

```diff
@@ -75,3 +75,114 @@ or, you can use the output from the status command above and run it through the
 commcare-cloud <env> ssh <machine>
 $ sudo supervisorctl tail -f <supervisor process name>
 ```
+
+## One of the setup commands is showing...
+### `RequestError: socket.error: [Errno 111] Connection refused`
+
+This means that CouchDB is unreachable.
+
+
+#### Breakdown of a request to CouchDB
+
+Note: if you are running on a recommended single-machine setup,
+then you can ignore the host groups (denoted `[in brackets]`):
+all services will be running on the same machine.
+
+
+Requests to CouchDB are made over HTTP,
+and are normally routed the following way:
+1. They start at the originator of the request,
+   such as a Django web worker
+2. They are made to port 25984 on host `[couchdb_proxy]`,
+   which is served by the `nginx` web server, acting as a load balancer.
+3. `nginx` passed them through to one of the `couchdb2` nodes
+   (or _the_ `couchdb2` node if you have only one),
+   which handles the requests.
+
+```
+[webworkers] [couchdb2_proxy] [couchdb2]
+django  -->  nginx  -------->  couchdb2
+             port 25984        port 15984
+```
+
+The following table represents the general case
+and includes variables that may be overriding the default port values:
+
+|   |  host group  |  service  |  port (default value) |  port (variable name) |
+|---|--------------|-----------|-----------------------|-----------------------|
+| Originator | various | various |  |  |
+|  | ⇩ |  |  |
+| CouchDB Load Balancer | `[couchdb2_proxy]` | `nginx` | 25984 | `couchdb2_proxy_port` |
+|  | ⇩ |  |  |
+| CouchDB Node | `[couchdb2]` | `couchdb2` | 15984 | `couchdb2_port` |
+
+
+#### How to confirm the issue
+
+To confirm the issue, that django processes cannot reach CouchDB, run
+
+```bash
+commcare-cloud <env> django-manage check_services
```

Comment:
could add `couch` as a param to only check couchdb which is much faster than checking everything

Reply:
Side note - we should standardize these names (couch, couchdb, postgres, postgresql)

Reply:
Thanks will. Yeah, I was noticing that too :/